staff/models/batch.py

```diff
@@ -9,6 +9,14 @@
 from student.models.enrolment import Enrolment
 
 
+class BasicsBatchManager(models.Manager):
```

Comment:
Nice abstraction. What is a "batch manager"? Wondering if that should be explained somewhere, either in comment or documentation